07-hosting.Rmd

```diff
@@ -373,3 +373,49 @@ You can peruse this information but keep an eye on the `restarts` count for your
 Thankfully, pm2 automatically manages the log files from your underlying processes. If you ever need to check the log files of a service, you can just run `pm2 logs run-myfile`, where `myfile` is again the name of the service obtained from `pm2 list`. This command will show you the last few lines logged from your process, and then begin streaming any incoming log lines until you exit (`Ctrl-c`). 
 
 If you want a big-picture view of the health of your server and all the pm2 services, you can run `pm2 monit` which will show you a dashboard of the RAM and CPU usage of all your services.
+
+## systemd {#systemd}
+
+(These steps are tested on Ubuntu Server 18.04 LTS)
```

Comment:
Would be great to clarify where else this is or isn't expected to work. i.e. "You can check if your server is using systemd by ___" Otherwise, I fear we're opening up more support tickets from confused users on systems that aren't compatible.

Reply:
explained that this works on Linux only